Sources/GRInputField/UIKit/InputFieldView.swift

```diff
@@ -640,7 +641,12 @@ public extension InputFieldView {
     }
 
     var isSelected: Bool { textField.isSelected }
-
+    
+    func setupAppearancee(_ customAppearance: InputFieldAppearance) {
```

Comment:
`appearance` tu je s dvomi `e` na konci 😄

Reply:
pridal by som sem podporu pre `nil` hodnoty - pokial je `customAppearance` nil, aby sa appearance revertla naspat na tu globalnu. Moze si to najst use case + zbavis sa vo SwiftUI view-cku toho `if let`-u 😄